src/layers.jl

```diff
@@ -202,13 +216,13 @@ end
 function (l::AGNConvDEQ)(gr::AtomGraph)
     p,re = Flux.destructure(l.conv)
     # do one convolution to get initial guess
-    guess = l.conv(gr).features
+    guess = l.conv(gr).encoded_features
 
     f = function (dfeat,feat,p,t)
         input = gr
-        input.features = reshape(feat,size(guess))
+        input.encoded_features = reshape(feat,size(guess))
```

Comment:
Since `AtomGraph`s no longer have `encoded_features` field, I don't think this would work... (`gr` is an `AtomGraph`)

Reply:
oh yeah that was a find and replace haha, that whole thing is prohibitively slow at the moment anyway, but I'll fix it